mon-pix/app/styles/pages/_skill-review.scss

```
@@ -246,6 +246,44 @@
 }
 
 .skill-review-competence-stages-result {
+  &__title {
+    @extend %pix-title-s;
+
+    padding-bottom: $pix-spacing-m;
+    color: $pix-neutral-90;
+    font-weight: $font-medium;
+    font-family: $font-open-sans;
+  }
+
+  &__content {
+    padding-top: $pix-spacing-m;
+
+    &:nth-child(2) {
+      padding-top: 0;
+    }
+  }
+
+  &__area {
+    display: flex;
+    align-items: center;
+    padding-bottom: $pix-spacing-xs;
+
+    h3 {
+      @extend %pix-body-l;
+
+      color: $pix-neutral-90;
+      font-weight: $font-medium;
```

Je suggère

```suggestion
      @extend %pix-weight-medium;
```

Mais peut être que c'est moins bien ?

---

Le premier est plus explicite je trouve

---

Apparement, c'est plutôt %pix-body-weight-medium ! Je ne sais pas si la doc de storybook n'est pas à jour

---

🙏 ne pas utiliser de placeholder / mixin si c'est juste pour remplacer une variable.

Avec un placeholder, quelqu'un qui découvre le code sera obligé d'aller voir ce qui se cache derrière.
Ça vaut le coup uniquement pour éviter d'écrire plusieurs lignes qui vont ensemble (font-size, weight, line-height, ...)

---

Du coup autant virer ces placeholders de weight non ?